.github/workflows/brew-bump.yml

```diff
@@ -8,9 +8,9 @@ jobs:
   bump_formula:
     runs-on: ubuntu-24.04
     steps:
-      - if: github.event.pull_request.head.ref == 'main'
       - name: Update Homebrew Formula
+        if: ${{ !github.event.release.prerelease }}
         uses: dawidd6/action-homebrew-bump-formula@v4
         with:
           token: ${{ secrets.RELEASE_GITHUB_TOKEN }}
```

Comment:
looks like this flow does not work?

is there any fork to use this workflow?

relates to https://github.com/Homebrew/homebrew-core/pull/201713